test/k8s-integration/driver-config.go

```diff
@@ -127,13 +127,9 @@ func generateDriverConfigFile(testParams *testParameters) (string, error) {
 		snapshotClassName = "no-volumesnapshotclass"
 	}
 
-	caps = append(caps, "pvcDataSource")
+	//caps = append(caps, "pvcDataSource")
```

Comment:
Since you add the sc-extreme storage class you can make the capability depend on the storage class choice, like we do for regional a couple lines further down.